content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -383,21 +383,21 @@ rules:
   verbs: ["get", "list", "watch"]
 ```
 
-Allow reading/writing Deployments (at the HTTP level: objects with `"deployments"`
-in the resource part of their URL) in both the `"extensions"` and `"apps"` API groups:
+Allow reading/writing Events (at the HTTP level: objects with `"events"` in the
+resource part of their URL) in both the core `""` and `"events.k8s.io"` API groups:
```

Comment:
I'd use Deployment, in its GA API group (`apps`). People rarely patch Events.

Reply:
Thanks for letting me know that `events` is not a common resource. :sweat_smile:

I think this section ([Role examples](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#role-examples)) is about how to write rules instead of most used resources:

1. Reading `pods` in the core API group
    - The most simple one
2. R/W `deployments` in `extensions` and `apps` API group
    - Multiple API groups
3. Reading `pods` in core API group and R/W `jobs` in `batch` and `extensions`
    - Multiple rules
4. Allow reading a ConfigMap named `my-config`
    - Must be bound with a RoleBinding to limit to a single ConfigMap in a single namespace.
5. Allow reading the resource "nodes" in the core group
    - Because a Node is cluster-scoped, this must be in a ClusterRole bound with a ClusterRoleBinding to be effective.
6. Allow GET and POST requests to the non-resource endpoint
    - Example for `nonResourceURLs`

Since `extensions` is deprecated, I think it's good to replace it with another (although not that common) example for multiple API groups.

There are only three resources that appeared in multiple API groups:
```
NAME      APIVERSION                NAMESPACED    KIND
events    v1                        true          Event
events    events.k8s.io/v1          true          Event
nodes     v1                        false         Node
nodes     metrics.k8s.io/v1beta1    false         NodeMetrics
pods      v1                        true          Pod
pods      metrics.k8s.io/v1beta1    true          PodMetrics
```

I am still a beginner at Kubernetes, could you give me some advice about which could be a better example for this page?

Reply:
The `nodes` and `pods` in v1 and metrics API groups are not the same resource, so I wouldn't expect to grant access to both in the same rule.

`events` are the only remaining built-in resource that appears in multiple API groups (we had a lot in the `extensions` API group but those got migrated to dedicated groups and the `extensions` API group was deprecated and disabled)

Reply:
Thanks for clarifying! Now I think it's better to change `events` example back to `deployments`. :blush: